Submitted By: DJ Lucas <dj_AT_linuxfromscratch_DOT_org>
Date: 2006-04-22
Initial Package Version: 2.0.2
Origin: self
Description: Fixes compilation with xorg-6.9/7.0
Upstream Status: submitted

--- OOB680_m5-orig/vcl/util/makefile.mk	2005-12-19 11:21:56.000000000 -0600
+++ OOB680_m5/vcl/util/makefile.mk	2006-03-26 22:28:25.000000000 -0600
@@ -318,7 +318,7 @@
 SHL2STDLIBS+=$(LIBSN_LIBS)
 .ENDIF
 
-SHL2STDLIBS += -lXext -lSM -lICE -lX11
+SHL2STDLIBS += -lXext -lSM -lICE -lX11 -lXau
 .IF "$(OS)"!="MACOSX" && "$(OS)"!="FREEBSD"
 # needed by salprnpsp.cxx
 SHL2STDLIBS+= -ldl
